tests/alive-tv/lifetime/lifetime-noub.srctgt.ll

```diff
@@ -12,3 +23,4 @@ define void @f() {
   unreachable
 }
 
+; ERROR: Source is more defined than target
```

Comment:
This is a non-functional change; just merges lifetime-noub.src.ll/.tgt.ll into lifetime-noub.srctgt.ll